frontend/packages/cli/src/cli/initCommand/index.ts

```diff
@@ -1,20 +1,149 @@
+
+💡 Have feedback? Share it with us!  
+Visit ${yocto.blueBright('https://github.com/liam-hq/liam/discussions')} to submit ideas or report issues.
+
+🌟️ ${yocto.bold('Love Liam ERD')}? Help us grow by starring our GitHub repository:  
+${yocto.blueBright('https://github.com/liam-hq/liam')}
+
+----
+
+Now, let’s get started with setting up your Liam ERD project.
+  `)
+
+  const answers = await inquirer.prompt([
+    {
+      type: 'list',
+      name: 'dbOrOrm',
+      message: 'Which Technology (database or ORM) are you using?',
+      choices: [
+        'PostgreSQL',
+        'Ruby on Rails (schema.rb)',
+        'Prisma',
+        'Drizzle',
+        'MySQL',
+      ],
+      default: 'PostgreSQL',
+    },
+    {
+      type: 'confirm',
+      name: 'canUsePgDump',
+      message:
+        'Can we use an .sql file generated by pg_dump? (Recommended: `--schema-only`)',
+      default: false,
+      when: (prev) => prev.dbOrOrm === 'PostgreSQL',
+    },
+    {
+      type: 'confirm',
+      name: 'addGhActions',
+      message: 'Generate GitHub Actions Workflow?',
+      default: false,
+    },
+  ])
+
+  const selectedFormat = formatMap[answers.dbOrOrm] || 'postgresql'
+
+  console.info('\n--- Summary of your choices ---')
+  console.info(`Database/ORM: ${answers.dbOrOrm}`)
+  if (answers.dbOrOrm === 'PostgreSQL') {
+    if (answers.canUsePgDump) {
+      console.info(
+        `Use an .sql file generated by pg_dump (recommended: --schema-only)? : ${
+          answers.canUsePgDump ? 'Yes' : 'No'
+        }`,
+      )
+    }
+  }
   console.info(
-    green(
-      bold(`
-    👾  Welcome to the @liam-hq/cli setup process! 👾
+    `Generate GitHub Actions Workflow?: ${answers.addGhActions ? 'Yes' : 'No'}`,
+  )
+
+  console.info(`
+For more details about Liam ERD usage and advanced configurations, check out:
+${yocto.blueBright('https://liambx.com/docs')}
+`)
 
-    This \`init\` subcommand will guide you interactively through the setup.
-    `),
+  console.info('\n--- Output ---')
+  console.info('\n--- Next Steps ---')
+
+  console.info(
+    '1) Build your ERD from a schema file (or from the dump.sql) using the following command:',
+  )
+  console.info(
+    yocto.blueBright(
+      `   $ liam erd build --input <schemafile> --format ${selectedFormat}`,
     ),
   )
+
+  console.info('\n2) Start your favorite httpd for serving dist. e.g.:')
+  console.info(yocto.blueBright('   $ npx http-server dist'))
+
+  if (answers.addGhActions) {
+    const workflowContent = `name: ERD Build
+on:
+  push:
+    branches: [ "main" ]
+
+jobs:
+  build-erd:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Generate ER Diagrams
+        run: npx @liam-hq/cli erd build --input <schemafile> --format ${selectedFormat}
+    
+    # - Next step: Add a step to upload your ERD \`./dist\` to share within your company
```

Comment:
It looked good to include the word "deploy to a hosting service":

```suggestion
    # - Next step: Deploy ERD `./dist` to your preferred hosting service for easy sharing and access.
```

Reply:
@hoshinotsuyoshi

Reply:
thanks!